src/6model/reprs/P6bigint.c

```diff
@@ -92,10 +89,10 @@ static MVMint64 get_int(MVMThreadContext *tc, MVMSTable *st, MVMObject *root, vo
     if (MVM_BIGINT_IS_BIG(body)) {
         mp_int *i = body->u.bigint;
         if (MP_NEG == SIGN(i)) {
-            return -mp_get_int64(tc, i);
+            return -mp_get_int64(tc, i, 1);
```

Comment:
It would look better if there were a space between the `-` and the function to make it more visible.